src/malli/core.cljc

```diff
@@ -935,6 +948,32 @@
   ([?schema options]
    (-type (schema ?schema options))))
 
+(defn walk
+  ([?schema f]
+   (walk ?schema f nil))
+  ([?schema f options]
+   (-walk
+     (schema ?schema options)
+     (reify Walker
+       (-accept [_ s _ _] s)
+       (-inner [this s in options] (-walk s this in options))
+       (-outer [_ s c in options] (f s c in options)))
+     [] options)))
+
+(defn ^:no-doc find-first
```

Comment:
This seems like a useful / worth exporting into `clj-doc` - but maybe it should live under `malli.utils` like`merge`, `assoc`, etc?

Reply:
good point! Will move it to `malli.util` and make public & add to README.

Reply:
done